files/default/ckan_cli

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Call either 'ckan' or 'paster' with appropriate syntax,
+# depending on what is present.
+
+# This script is aware of the VIRTUAL_ENV environment variable, and will
+# attempt to respect it with similar behaviour to commands like 'pip'.
+# Eg placing this script in a virtualenv 'bin' directory will cause it
+# to call the 'ckan' or 'paster' command in that directory, while
+# placing this script elsewhere will cause it to rely on the VIRTUAL_ENV
+# variable.
+
+CKAN_INI_FILE="${CKAN_INI_FILE:-/etc/ckan/default/production.ini}"
```

Comment:
Please add more info that this will use Environment variables or default to /etc/ckan/default/production.ini  as all places that this is now used looks better that also is a major change from what it had before.